### PR TITLE
Merge final release/19.0 into trunk

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] Version bumped in `*.xcconfig` files
 - [x] `*.lproj/Localizable.strings` files updated with latest translations
 - [x] `fastlane/metadata/*` and `fastlane/jetpack_metadata/*` files updated with latest translations
 - [x] WPiOS submitted to Apple for review
 - [x] JPiOS submitted to Apple for review

Compared to the last beta, this diff adds those last-minute additions:
 - https://github.com/wordpress-mobile/WordPress-iOS/pull/17793
 - https://github.com/wordpress-mobile/WordPress-iOS/pull/17798

### CODE REVIEW

**Comment by mokagio** [comment]
Unit tests [failed on CircleCI](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-iOS/26984/workflows/d0ad1016-2263-475d-9eb3-cd8795024988/jobs/64436?invite=true#step-106-3882) with `PostServiceWPComTests.testTrashingAPostWillUpdateItsRevisionStatusAfterSyncProperty()`. All green on Buildkite.

I restarted the build (accidentally doing "rerun workflow" instead of "rerun from failed"). I've been experiencing some flakiness on CircleCI recently. Haven't put much weight on it because we plan to decommission it soon™.

**Comment by AliSoftware** [comment]
@mokagio the CI workflow running the tests also interrupted/cancelled the Release Build, which I had to restart, and might also explain why the Unit Test workflow were then cancelled in turn.

It happens that the Release Build (the one I relaunched after ☝️) ultimately failed anyway, because the new App Icon for AppStore got rejected for having an alpha channel (see Slack). Which means @jkmassel (who is now taking over the submission as I'm EOD) will need to update it with a new one we got without alpha, and restart the Release Build CI workflow after that.

All that meaning that the Unit Test workflow that you just restarted… might be cancelled and fail again as soon as Jeremy pushes a new commit with the asset fix, then re-triggers a Release Build (which will take priority obviously).

TL;DR: We should probably wait until after the Release Build workflow on CI finally goes green and we successfully submitted the builds to ASC before restarting the Unit Test jobs on that PR 🙃

File: config/Version.internal.xcconfig [hunk 1]
```diff
@@ -1,9 +1,4 @@
 VERSION_SHORT=19.0
 
 // Internal long version example: VERSION_LONG=9.9.0.20180423
-//
-// Technically, the date for this version was 10/01, but this is the second
-// build for the day so I bumped it to 10/02. On 10/04 we'll code freeze 18.4
-// and this should be the very last build for 18.3, so we should be fine.
-// – Gio
-VERSION_LONG=19.0.0.20220121
+VERSION_LONG=19.0.0.20220122
```
Comment (AliSoftware) [inline config/Version.internal.xcconfig:4]: > Also notice that I bumped the internal version to "...22", which
technically would mean the build was made on the 22nd of January, even
though (as far as UTC goes) we're still on the 21st. Pragmatically, that
has little consequence because we'll most likely won't be making any
build tomorrow.

Can't wait for us to address https://github.com/wordpress-mobile/release-toolkit/issues/202 one day…

**Review by mokagio — APPROVED** [review]
